Header header

LocMapLocation[] locations
string[] groups